src/node_buffer.cc

```diff
@@ -1373,6 +1388,7 @@ void Initialize(Local<Object> target,
   SetMethodNoSideEffect(context, target, "encodeUtf8String", EncodeUtf8String);
 
   SetMethodNoSideEffect(context, target, "isUtf8", IsUtf8);
+  SetMethodNoSideEffect(context, target, "isAscii", IsAscii);
```

Comment:
Kind of applies to both of these, but should these have fast API call implementations? I don't think there's anything speaking against that, right?

Reply:
The only fast path is for `FastApiTypedArray`. If your comment is a non-blocker, I'd like to invest on it, after this pull request is landed.